src/api/process.c

```diff
@@ -23,9 +23,22 @@
 #define READ_BUF_SIZE 2048
 
 #if _WIN32
-typedef HANDLE process_handle;
+
+typedef DWORD process_error_t;
+typedef HANDLE process_handle_t;
```

Comment:
do note that the `_t` suffix on types is reserved by posix.
(despite that its commonly done, so I don't think its worth more than a note)

Reply:
wow, I didn't know that!

For future reference, what are other commonly used suffix/prefix for non-standard libraries?

Reply:
Commonly used suffixes are:
- None at all e.g. `process_error`
- `_t` suffix e.g. `process_error_t`
- `_s` suffix for structs (not applicable in this case)
- Project name prefix e.g. `LXL_process_error`
possibly others.

Its not a big deal, and many people ignore it.